tests/unit/module_utils/test_ec2.py

```diff
@@ -76,3 +79,20 @@ def test_ansible_dict_with_integer_to_boto3_filter_list(self):
 
         converted_filters_int = ansible_dict_to_boto3_filter_list(filters)
         self.assertEqual(converted_filters_int, filter_list_integer)
+
+    # ========================================================
+    #   ec2.is_outposts_arn
+    # ========================================================
+
+    @parametrize(
```

Comment:
Use `@pytest.mark.parametrize` instead.

Reply:
@goneri  @pytest.mark.parametrize cannot be used with unittest.TestCase methods, for this reason Mandar use parametrize.